lti_consumer/utils.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"""
+Make '_' a no-op so we can scrape strings
```

Comment:
Perhaps I am not understanding -- where is the tie-in to self.ugettext for returning the actual translated strings if the implementation references this no-op?